src/commands/blobs/blobs-delete.mjs

```diff
@@ -0,0 +1,43 @@
+// @ts-check
+import { getStore } from '@netlify/blobs'
```

Comment:
[question] can the CLI commands be used with deploy-specific stores?

Reply:
No.